packages/web-reporter/writeReport.ts

```diff
@@ -1,11 +1,44 @@
 import fs from "fs";
+import { TestCaseResult } from "@perf-profiler/types";
+
+// We should have better management of the time interval over the repo
+const MEASURE_INTERVAL = 500;
+
+export const getMeasuresForTimeInterval = ({
+  duration,
+  skip,
+  results,
+}: {
+  duration: number | null;
+  skip: number;
+  results: TestCaseResult[];
+}): TestCaseResult[] => {
+  const firstMeasureIndex = skip / MEASURE_INTERVAL;
```

Comment:
If skip === 300, the first index will be 0.6.

We should use Math.ceil on the result